### PR TITLE
Add "suggested docker images" tree view

### PR DESCRIPTION
**Pull request by LazarusX**

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -94,9 +97,22 @@ export function activate(context: vscode.ExtensionContext) {
         dockerHubTreeDataProvider.pullFromHub(element.parent, element.name);
     }));
 
+    context.subscriptions.push(vscode.commands.registerCommand("docker-explorer.pullLatestFromDockerHub", (repo) => {
+        suggestedDockerImages.pullFromHub(repo.user, repo.repository);
+    }));
+
+    context.subscriptions.push(vscode.commands.registerCommand("docker-explorer.openDockerHubPage", (repo) => {
```
Comment (formulahendry) [inline src/extension.ts]: Track AI event

File: package.json [hunk 2]
```diff
@@ -204,6 +220,21 @@
                     "command": "docker-explorer.pullFromDockerHub",
                     "when": "view == DockerHubTreeView && viewItem == image",
                     "group": "docker-explorer@0"
+                },
+                {
+                    "command": "docker-explorer.openDockerHubPage",
+                    "when": "view == DockerHubTreeView && viewItem == image",
+                    "group": "docker-explorer@0"
```
Comment (formulahendry) [inline package.json]: docker-explorer@1

File: package.json [hunk 3]
```diff
@@ -204,6 +220,21 @@
...
+                },
+                {
+                    "command": "docker-explorer.pullLatestFromDockerHub",
+                    "when": "view == suggestedDockerImages",
+                    "group": "docker-explorer@0"
+                },
+                {
+                    "command": "docker-explorer.openDockerHubPage",
+                    "when": "view == suggestedDockerImages",
+                    "group": "docker-explorer@0"
```
Comment (formulahendry) [inline package.json]: docker-explorer@1

File: src/suggestedDockerImages.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                DockerTreeBase.isErrorMessageShown = true;
+            }
+        }
+
+        return images;
+    }
+
+    public pullFromHub(user: string, repository: string): void {
+        Executor.runInTerminal(`docker pull ${user == null ? repository : user + "/" + repository}`);
+        AppInsightsClient.sendEvent("pullSuggestImageFromHub");
```
Comment (formulahendry) [inline src/suggestedDockerImages.ts]: pullSuggestedImageFromHub

**Comment by LazarusX** [comment]
@formulahendry Thanks for the comment. The issues have been resolved.

**Pull request merged**